arangod/VocBase/Methods/Indexes.cpp

```diff
@@ -752,10 +756,16 @@ Result Indexes::drop(LogicalCollection& collection,
 
     transaction::Options trxOpts;
     trxOpts.requiresReplication = false;
+#ifdef USE_V8
     SingleCollectionTransaction trx(transaction::V8Context::createWhenRequired(
```

Comment:
Why is V8context used when V8 is available?

Reply:
I don't know right now. In this PR I simply wanted to preserve the original code for the case we build with V8.

Reply:
Ok